tests/assertions/is_substring_of.zsh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env zunit
+
```

Comment:
it seems like we cant make this work with the assert function anyway.

`assert "" is_substring 'some string'` treats the command as `assert is_substring 'some_string'` because the string is empty